src/pages/profile.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import AvatarImage from '../components/AvatarImage';
+import NavBar from "../components/NavBar";
+import "../css/Article.css";
+import "../css/Profile.css";
+import {Paper} from "material-ui";
+import Container from "@material-ui/core/Container";
+import SimpleList from "../components/SimpleList";
+import Button from "@material-ui/core/Button";
+import IconsList from "../components/IconsList";
+import {Looks3, LooksOne, LooksTwo} from "@material-ui/icons";
+
+export default class ProfilePage extends React.Component {
+    render() {
+        const imageStyle = {
+            gridColumn: "1/span 7"
+        };
+        return (
+            <React.Fragment>
+                <NavBar/>
+                <Container className={"profile"}>
+                    <Paper>
+                        <article>
+                            <div className={"flexbox"}>
+                            <span><AvatarImage large={true} variant={"rounded"} className={"ProfilePicture"}
```

Comment:
For containing several elements use block-elements instead of `span`'s.

Reply:
Fixed